kube-derive/src/custom_resource.rs

```diff
@@ -808,28 +808,28 @@ struct StatusInformation {
 /// returns: A `StatusInformation` struct
 fn process_status(
     root_ident: &Ident,
-    status: &Option<String>,
+    status: &Option<Path>,
     visibility: &Visibility,
     kube_core: &Path,
 ) -> StatusInformation {
-    if let Some(status_name) = &status {
-        let ident = format_ident!("{}", status_name);
+    if let Some(status_path) = &status {
+        let pth = quote! { #status_path };
```

Comment:
```suggestion
    if let Some(pth) = &status {
```

No need for this `quote!`, since we just splice it into other quotes further down anyway, which would've done the same.

Reply:
ah, thanks!